client/src/templates/Challenges/redux/completion-epic.js

```diff
@@ -2,14 +2,7 @@ import { navigate } from 'gatsby';
 import { omit } from 'lodash-es';
 import { ofType } from 'redux-observable';
 import { of, empty } from 'rxjs';
-import {
-  switchMap,
-  retry,
-  catchError,
-  concat,
-  filter,
-  finalize
-} from 'rxjs/operators';
+import { switchMap, retry, catchError, concat, map } from 'rxjs/operators';
```

Comment:
```suggestion
import { switchMap, retry, catchError, concat, tap } from 'rxjs/operators';
```
`tap` is fine, since we're not transforming anything.